modules/nodes-containers-start-pod-safe-sysctls.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+//
+// * nodes/containers/nodes-containers-sysctls.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-starting-pod-safe-sysctls_{context}"]
+= Starting a pod with safe sysctls
+
+You can set sysctls on pods using the pod's `securityContext`. The `securityContext` applies to all containers in the same pod.
```

Comment:
You can set sysctls on pods using the pod's 
=>
Sysctl's can be set ....

Reply:
funny my linter on the .adoc file complains as this is passive voice. We try to maintain active voice.